storage/src/tests/persistence/processalltest.cpp

```diff
@@ -101,7 +105,8 @@ TEST_F(ProcessAllHandlerTest, remove_location_throws_exception_on_bogus_selectio
     document::Bucket bucket = makeDocumentBucket(bucketId);
     auto cmd = std::make_shared<api::RemoveLocationCommand>("id.bogus != badgers", bucket);
 
-    ProcessAllHandler handler(getEnv(), getPersistenceProvider());
+    document::BucketIdFactory bucketIdFactory;
```

Comment:
Consider moving the `BucketIdFactory` up into the `ProcessAllHandlerClass` test fixture class

Reply:
Fixed